cirq/sim/mux.py

```diff
@@ -52,6 +52,53 @@ def sample(program: Union[circuits.Circuit, schedules.Schedule],
             repetitions=repetitions)
 
 
+def out_vector(program: Union[ops.Operation, ops.Gate, circuits.Circuit, schedules.Schedule],
+               *,
+               initial_state: Union[int, np.ndarray] = 0,
+               param_resolver: Optional[study.ParamResolver] = None,
+               qubit_order: ops.QubitOrderOrList = ops.QubitOrder.DEFAULT,
+               dtype: Type[np.number] = np.complex64
+               ) -> 'np.ndarray':
+    """Returns the wavefunction resulting from applying an object.
+
+    Args:
+        program: The circuit, schedule, gate, or operation to apply to the
+            initial state in order to produce the result.
+        param_resolver: Parameters to run with the program.
+        qubit_order: Determines the canonical ordering of the qubits. This
+            is often used in specifying the initial state, i.e. the
+            ordering of the computational basis states.
+        initial_state: If an int, the state is set to the computational
+            basis state corresponding to this state. Otherwise  if this
+            is a np.ndarray it is the full initial state. In this case it
+            must be the correct size, be normalized (an L2 norm of 1), and
+            be safely castable to an appropriate dtype for the simulator.
+        dtype: The `numpy.dtype` used by the simulation. Typically one of
+            `numpy.complex64` or `numpy.complex128`.
+            Favors speed over precision by default, i.e. uses `numpy.complex64`.
+    """
```

Comment:
doc return value. note the order for the gate case.

Reply:
Done.